media/css/mozorg/home/home-b.less

```diff
@@ -0,0 +1,1160 @@
+            .flex-direction(column);
+            .flexbox;
+            .justify-content(stretch);
+            float: none;
+        }
+
+        .slider-cover,
+        .slider-detail {
+            .flex;
+            height: @sliderHeight;
+        }
+
+        .slider-cover {
+            .align-items(center);
+            .flex-direction(column);
+            .flexbox;
+            .justify-content(center);
+
+            .slider-content p {
+                .font-size(20px);
+            }
+        }
+
+        .slider-detail {
+            min-height: 0;
+        }
+
+        // if JS available, set up fancy slider interactions
+        .js {
+            .toggle-content {
+                display: block;
+                height: 44px;
+                overflow: hidden;
+            }
+
+            .toggle-buttons {
+                transition: transform 0.25s ease;
+            }
+
+            .toggle-close,
+            .toggle-open {
+                background: transparent;
+                border: none;
+                cursor: pointer;
+                display: block;
+                height: 40px;
+                outline: none;
+            }
+
+            .toggle-open {
+                .font-size(26px);
+                font-weight: bold;
+                height: 40px;
+                margin: 0 auto 4px;
+                text-decoration: underline;
+            }
+
+            .toggle-close {
+                background: url('/media/img/home/new/b/svg-sprite.svg');
+                cursor: pointer;
+                height: 44px;
+                margin: 0 auto;
+                overflow: hidden;
+                transition: transform 0.2s ease;
+                width: 44px;
+
+                span {
+                    visibility: hidden;
+                }
+            }
+
+            .icon-links {
+                width: 50%;
+
+                li {
+                    margin-bottom: 12px;
+                }
+            }
+
+            // update icons to alternate color
+            #encryption a {
+                .icon-link(43px, -229px);
+            }
+
+            #teachtheweb a {
+                .icon-link(32px, -272px);
+            }
+
+            #leandata a {
+                .icon-link(34px, -304px);
+            }
+
+            #gaming a {
+                .icon-link(34px, -338px);
+            }
+
+            #vr a {
+                .icon-link(23px, -372px);
+            }
+
+            #servo a {
+                .icon-link(30px, -395px);
+            }
+
+            #rust a {
+                .icon-link(35px, -425px);
+            }
+
+            #who-innovate-wrapper {
+                .clearfix;
+                display: block;
+                height: @sliderHeight;
+                overflow: hidden;
+                position: relative;
+                width: 100%;
+
+                &.open-who {
+                    #innovate {
+                        transform: translateX(100%);
```

Comment:
Yeah, it's a good question. I didn't use the mixins here, as the rules are nested in a `@supports (display: flex)`, which I'm taking as good enough to support un-prefixed transitions and transforms (at least on evergreen browsers).

Looks like the Android browser would be the primary edge case here - seems to support flexbox in the last 3 versions, but the previous 2 still need prefixes on transforms.

I'm of your mindset - as long as it's a progressive enhancement, let's not worry about lagging browsers/old vendor prefixes. Going to leave as-is here.